### PR TITLE
databases: add locale and template parameters to DB creation

### PR DESCRIPTION
**Pull request by yakimant**
https://github.com/status-im/infra-logos-office/issues/31

### CODE REVIEW

File: templates/init/databases.sql [hunk 1]
```diff
@@ -3,5 +3,14 @@
...
+
+CREATE DATABASE "{{ db.name }}"
+WITH OWNER "{{ db.get("user", db.name) }}"
+{% if 'locale' in db %}
+LOCALE "{{ db.locale }}"
+{% endif %}
+{% if 'template' in db %}
+TEMPLATE "{{ db.template }}"
+{% endif %}
+;
```
Comment (siddarthkay) [inline templates/init/databases.sql:15]: nit pick : semi colon on new line looks weird :D 
```suggestion
{% endif %};
```

**Review by siddarthkay — APPROVED** [review]
nice

**Review by apentori — APPROVED** [review]

File: templates/init/databases.sql (hunk 1 above)
Comment (yakimant) [reply templates/init/databases.sql:15]: I think it's fine for sql - commands are multiline naturally.
And optional parameters are split with mandatory semicolon.
What do you think?

Comment (siddarthkay) [reply templates/init/databases.sql:15]: oh right I did not consider the optional condition.
your existing way seems better.

Thank you.

**Review by jakubgs — COMMENTED** [review]
You should have updated the `defauts/main.yml` and `README.md` with info about these options being available.

**Pull request merged**